components/v1/Navigation.tsx

```diff
@@ -21,24 +21,24 @@ type NavigationProps = {
 const Navigation: FunctionComponent<NavigationProps> = ({ marketsActive, portfolioActive }) => {
     return (
         <div className="container z-10 mx-auto max-w-full sm:z-20">
-            <div className="grid grid-cols-3 place-items-center content-center p-4">
-                <div className="justify-self-start">
+            <div className="flex flex-row items-center p-4 sm:space-x-16">
```

Comment:
@math-marcellino this should be `48px` or `space-x-12` not `64px`, please check the figma by @ayikfour